app/workers/commit_monitor_handlers/commit/bugzilla_checker.rb

```diff
@@ -0,0 +1,112 @@
+module CommitMonitorHandlers
```

Comment:
I like that idea.  I originally renamed the class because I realised it was doing more now than just commenting, but splitting them out seems like a cleaner solution.  Especially if we want to add more Bugzilla functionality in the future.